src/collectors/aws/redshift/ClusterSecurityGroups.ts

```diff
@@ -0,0 +1,39 @@
+import * as AWS from "aws-sdk";
```

Comment:
Please follow other file name. First letter should be small.

Reply:
ClusterSecurityGroups.ts --> clusterSecurityGroups.ts

Reply:
The necessary changes have been done. Please have a look at the code.

Reply:
Please check the modified versions of the codes.